src/views/Search.vue

```diff
@@ -5,22 +5,29 @@
         <search-form />
       </v-col>
     </v-row>
-    <v-row class="justify-end" style="margin-bottom: -10px">
-      <v-col sm="4" md="2" class="py-1">
-        <v-select
-          v-model="filter_sort"
-          :items="options.sort"
-          dense
-          :label="$t('views.search.sortByLabel')"
-        />
-      </v-col>
-      <v-col sm="4" md="2" class="py-1">
-        <v-select
-          v-model="filter_type"
-          :items="options.type"
-          dense
-          :label="$t('views.search.typeDropdownLabel')"
-        />
+    <v-row class="justify-center">
+      <v-col
```

Comment:
Unsure about this one, there's a vcol in a vcol now? I'll have to look at it when built.

Reply:
Currently at the xl breakpoint, the two dropdowns go to the far edge of the screen, and not in line with the results anymore. This keeps it in line with the results at the xl breakpoint. Basically it has the same breakpoint as the results.

Reply:
oh wait, shoot, we entirely redid the search component (lol) sorry